### PR TITLE
Feature/doorkeeper locale

### PR DESCRIPTION
**Pull request by berkes**
Doorkeeper locale was outdated. 

This caused "missing translation messges" in doorkeeper errors:

afbeelding

This PR attempts to sync with upstream en.yml, without loosing our customisations.

### CODE REVIEW

File: config/locales/doorkeeper.en.yml [hunk 1]
```diff
@@ -111,41 +121,10 @@ en:
       admin:
         nav:
           applications: Applications
+          home: Home
           oauth2_provider: OAuth2 Provider
+        title: Doorkeeper
       application:
         title: OAuth authorization required
-    scopes:
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Removing necessary strings

File: config/locales/doorkeeper.en.yml [hunk 2]
```diff
@@ -60,37 +64,43 @@ en:
...
+        title: Authorization code
     authorized_applications:
       buttons:
         revoke: Revoke
       confirmations:
         revoke: Are you sure?
       index:
         application: Application
-        created_at: Authorized
+        created_at: Created At
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 3]
```diff
@@ -60,37 +64,43 @@ en:
...
         title: An error has occurred
       new:
-        able_to: It will be able to
-        prompt: Application %{client_name} requests access to your account
+        able_to: This application will be able to
+        prompt: Authorize %{client_name} to use your account?
         title: Authorization required
       show:
-        title: Copy this authorization code and paste it to the application.
+        title: Authorization code
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 4]
```diff
@@ -60,37 +64,43 @@ en:
       error:
         title: An error has occurred
       new:
-        able_to: It will be able to
-        prompt: Application %{client_name} requests access to your account
+        able_to: This application will be able to
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 5]
```diff
@@ -60,37 +64,43 @@ en:
...
+        prompt: Authorize %{client_name} to use your account?
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 6]
```diff
@@ -31,27 +33,29 @@ en:
...
       show:
         actions: Actions
-        application_id: Client key
-        callback_urls: Callback URLs
+        application_id: Application UID
+        callback_urls: Callback urls
+        confidential: Confidential
         scopes: Scopes
-        secret: Client secret
+        secret: Secret
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 7]
```diff
@@ -31,27 +33,29 @@ en:
...
+        new: New Application
         title: Your applications
       new:
-        title: New application
+        title: New Application
       show:
         actions: Actions
-        application_id: Client key
-        callback_urls: Callback URLs
+        application_id: Application UID
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 8]
```diff
@@ -31,27 +33,29 @@ en:
...
+        callback_urls: Callback urls
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 9]
```diff
@@ -31,27 +33,29 @@ en:
...
+          'yes': 'Yes'
         name: Name
-        new: New application
-        scopes: Scopes
-        show: Show
+        new: New Application
         title: Your applications
       new:
-        title: New application
+        title: New Application
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

File: config/locales/doorkeeper.en.yml [hunk 10]
```diff
@@ -3,19 +3,21 @@ en:
   activerecord:
     attributes:
       doorkeeper/application:
-        name: Application name
+        name: Name
         redirect_uri: Redirect URI
-        scopes: Scopes
-        website: Application website
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Removing necessary strings

File: config/locales/doorkeeper.en.yml [hunk 11]
```diff
@@ -3,19 +3,21 @@ en:
   activerecord:
     attributes:
       doorkeeper/application:
-        name: Application name
+        name: Name
         redirect_uri: Redirect URI
-        scopes: Scopes
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Removing necessary strings

File: config/locales/doorkeeper.en.yml [hunk 12]
```diff
@@ -3,19 +3,21 @@ en:
   activerecord:
     attributes:
       doorkeeper/application:
-        name: Application name
+        name: Name
```
Comment (Gargron) [inline config/locales/doorkeeper.en.yml]: Keep old

**Review by Gargron — CHANGES_REQUESTED** [review]

**Comment by berkes** [comment]
Please let me know if you'd rather have this PR rebased & force pushed. Or if you are OK with separate commits for the requested fixes.

**Comment by SuperSandro2000** [comment]
Bump

**Comment by berkes** [comment]
@SuperSandro2000 anything else I can or should do, other than occasionally rebasing on master?

**Comment by SuperSandro2000** [comment]
IDK. I just keep spam bot away from things that are not spam.

**Pull request closed without merging**